tests/test_table.py

```diff
@@ -1535,23 +1535,18 @@ def test_bool_column(self, reset_db: None, reload_tester: ReloadTester) -> None:
         assert len(res) == 4
         assert res['col_0'] == [False, True, True, True]
 
+        # test adding a bool column with a computed value
+        t1.add_column(bool_computed=t1.c1 > 1)
+        res = t1.collect()
```

Comment:
Use the computed column in a where(), eg, t1.where(t.bool_computed) and t1.where(~t1.bool_computed).

Also, add these to the reload tests, unless there's a specific reason not to do that.

Reply:
Done.
<!-- Reviewable comment -ODrgbvt2-XqD_NRYjxP:b-896fix -->
<!-- Sent from Reviewable.io -->